notifications.yaml

```diff
@@ -0,0 +1,242 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Notification API
+  description: Notification API.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/messages/{id}':
+    x-swagger-router-controller: Messages
+    parameters:
+      - name: id
+        in: path
+        type: string
+        required: true
+        description: 'The message ID, as returned from the POST operation (submitMessageFromUser).'
+    get:
+      operationId: getMessage
+      summary: Get Message
+      description: The previously created message with the provided message ID is returned.
+      responses:
+        '200':
+          description: Message found.
+          schema:
+            $ref: '#/definitions/CreatedMessage'
+          examples: {}
+        '404':
+          description: No message found for the provided ID.
+  '/users/{fiscal_code}/messages':
+    x-swagger-router-controller: Messages
+    parameters:
+      - $ref: '#/parameters/PaginationRequest'
+      - name: fiscal_code
+        in: path
+        type: string
+        required: true
+        description: 'The fiscal code of the user, all upper case.'
+        maxLength: 16
+        minLength: 16
+        pattern: '[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]'
+    get:
+      operationId: getMessagesByUser
+      summary: Get messages by user
+      description: |-
+        Returns the messages for the user identified by the provided fiscal code.
+        Messages will be returned in inverse submission order (from last to first).
+        The "next" field, when present, contains an URL pointing to the next page with the other results in the collection.
+      responses:
+        '200':
+          description: Found.
+          schema:
+            allOf:
+              - type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/definitions/CreatedMessage'
```

Comment:
> For this reason

you mean just to have a shorter payload in the response ? 
maybe it's worth to have at least body_short to avoid a N+1 select problem.